apps/vscode/editor/src/utils/externalUrlContentHandler.ts

```diff
@@ -0,0 +1,25 @@
+import { Editor } from 'tldraw'
+import { rpc } from './rpc'
+
+export function registerExternalUrlContentHandler(editor: Editor) {
+	const currentFilesContentHandler = editor.externalContentHandlers['files']
+	const currentUrlContentHandler = editor.externalContentHandlers['url']
+	editor.registerExternalContentHandler('url', async (info) => {
+		// This happens when you shift drag a file from the file explorer to the editor
+		if (info.url.startsWith('file://')) {
```

Comment:
can it be multiple files?

Reply:
I couldn't drag multiple files in my testing. We also don't support `urls` external handlers, only a singular `url`.

Reply:
i wasn't sure if you could press `shift` and select multiple files / drag them that way